logging/fluentd-daemonset/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM fluent/fluentd:v1.2.1-debian-onbuild
+MAINTAINER Petr Kotas <petr.kotas@gmail.com>
+
+RUN buildDeps="sudo make gcc g++ libc-dev ruby-dev" \
+ && apt-get update \
+ && apt-get install -y --no-install-recommends $buildDeps \
+ && sudo gem install \
+        fluent-plugin-secure-forward \
+        fluent-plugin-record-reformer \
+        fluent-plugin-kubernetes_metadata_filter \
+        ffi \
+        fluent-plugin-systemd \
+        fluent-plugin-record-reformer \
```

Comment:
- Roman mentioned this earlier: what is `fluent-plugin-secure-forward` for? Seems it's not used, I don't see configuration for it. And I think we just don't need it, why would we want to encrypt logs in dev/ci?
- `fluent-plugin-record-reformer` also isn't used (and it's mentioned twice)
- the other plugins are in the `fluent/fluentd-kubernetes-daemonset:v1.2-debian-syslog` docker image, can't we just use that one and override config?

Reply:
Yep, we can remove the forwarder. Doing that.

As for the `fluent/fluentd-kubernetes-daemonset:v1.2-debian-syslog`, I double checked and there really is no need to use custom build.  Will remove it, use the default image and override configs.